Makefile

```
@@ -120,8 +120,8 @@ clean: ## Clean tmp, _output dirs and go cache/testcache.
 	go clean -cache -testcache ./...
 
 .PHONY: fmt
-fmt: $(GOFUMPORTS) ## Run gofumpt against code.
-	@$(GOFUMPORTS) -l -w $(shell find internal apis controllers test version -name '*.go') ./*.go
+fmt: $(GOFUMPT) ## Run gofumpt against code.
+	@$(GOFUMPT) -l -w $(shell find internal apis controllers test version -name '*.go') ./*.go
```

It appears to me as if the new version of `gofumpt` uses a different parameter than the old version. It seems to work on the whole module instead of individual `.go` files, so just calling it with `.` seems to produce the same result without outputting all filenames to stdout (don't know why it does this):

```suggestion
	@$(GOFUMPT) -l -w .
```